### PR TITLE
Register PushInfo with django admin.

### PR DESCRIPTION
**Pull request by jamaalscarlett**
Create send_message test method.

### CODE REVIEW

**Comment by jamaalscarlett** [comment]
@safwanrahman ?

**Comment by safwanrahman** [comment]
@jamaalscarlett I will check this in this weekend. Sorry for the delay,

File: webpush/admin.py [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+from .utils import _send_notification
+
+
+class PushInfoAdmin(admin.ModelAdmin):
+    list_display = ("__str__", "user", "subscription", "group")
+    actions = ("send_message",)
+
+    def send_message(self, request, queryset):
+        result = []
+        payload = "{\"head\": \"Hey\", \"body\": \"Hello World\"}"
```
Comment (safwanrahman) [inline webpush/admin.py]: Can you please make a pyhton ``dictionary`` and then make it json with ``json.dump()``? So it will look more conventional.

**Comment by jamaalscarlett** [comment]
@safwanrahman changes have been made.

File: webpush/admin.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+
+class PushInfoAdmin(admin.ModelAdmin):
+    list_display = ("__str__", "user", "subscription", "group")
+    actions = ("send_test_message",)
+
+    def send_test_message(self, request, queryset):
+        result = []
+        payload = {"head": "Hey", "body": "Hello World"}
+        for device in queryset:
+            result.append(_send_notification(device, json.dumps(payload), '0'))
```
Comment (safwanrahman) [inline webpush/admin.py]: The ``ttl`` should be integer ``0``

**Comment by safwanrahman** [comment]
Thanks @jamaalscarlett 
It looks awesome other than the small nit.
r+
Fix that and I will merge

**Comment by jamaalscarlett** [comment]
@safwanrahman Thanks, I will fix it when I get home.

**Comment by jamaalscarlett** [comment]
@safwanrahman fixed

**Comment by safwanrahman** [comment]
@jamaalscarlett Looks good!
Thanks a lot for the PR

**Pull request merged**